openapi/adintel/adintel.yaml

```diff
@@ -133,6 +133,33 @@ paths:
       tags:
         - accountStats
         - options
+  '/connectedAccount/{connectedAccount.id}':
+    parameters:
+      - schema:
+          type: string
+        name: connectedAccount.id
+        in: path
+        required: true
+    options:
+      summary: 'List valid HTTP verbs for /connectedAccount/{id}'
+      operationId: options-connectedAccount-connectedAccount.id
+      responses:
+        '200':
+          description: OK
+      description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
+      tags:
+        - Connected Account
+  /connectedAccount:
+    parameters: []
+    options:
```

Comment:
You have OPTIONS operations but are missing GET operations for both of these paths.

Reply:
oh good catch! Seems to be coming from a previous merge conflict

Reply:
Should be fixed